pages/organizers.qmd

```diff
@@ -4,7 +4,7 @@
 #| label: organizer_table
 #| warning: false
 #| echo: false
-source(file.path("..", "R", "organizers.R"))
-create_organizer_table()
+#source(file.path("..", "R", "organizers.R"))
```

Comment:
I guess this removes the table completely? Instead, print one big table with all organizers (without any division to local or community)